### ISSUE TITLE
Upload Diego Ecuador Article

### ISSUE THREAD

**Issue by freddydrew** [freddydrew/fredducation#55]
One day at Parque Carolina there was this guy sitting in the grass next to his bike with some guys I had met a few days earlier and some kids. When I went over to say hello and introduce myself, he had a lunchbox open and was selling stickers to these kids. Ya gotta respect the hustle. We chatted for a bit and I was relieved to find out he speaks English very well. While we spoke Spanish most of the time, it was nice to have someone occasionally help me out when I'm confused. So we rode a bit, but mostly just hung out and did some people watching that first day. There was a guy on a mountain bike flying around the skate park and he narrowly avoided falling hard when jumping out of a ramp. We both saw this happen and naturally I yelled out *"the dude almost ate shit!"*. Diego laughed and told me he never heard that before. He said saying "*toma*" would be appropriate for such a situation, kind of like *take it* or *take the fall*. That was the first of many things my buddy taught me in spanish. Later on in New York I'd be watching playoff baseball and notice [Adolis García](https://en.wikipedia.org/wiki/Adolis_Garc%C3%ADa) yell *TOMA!* after crushing the ball. I thought of my friend Diego. 

Another day at the park he gave me the rundown on bicycle parts in Spanish, it felt like BMX anatomy. Volantes, la manzana, bielas, ruedas, llantas, pegs (ofcourse), and a bunch of other words were part of his vocabulary lesson that day. I'm sure there will be an "Anatomy of the BMX bike" post on this site in honor of my dude Diego. Later on that same day I got my first glimpse of this guy's skills on the bike and realized he's an animal. I jokingly said *this fuckin pro rider over here* and he cracked up. He had earned his nickname for sure. One day at the park he mentioned that he'll be heading to a friends house to help him work on a half pipe and that afterward they'd be going to a dirtpark. He said I should come along so I said yes. I had gone to [La Poma Bike park](https://www.instagram.com/lapomabikepark/?hl=en) years earlier but just sort of watched while my trail rider friends teared up the course. I have no business on dirt jumps but I figured why not, it always looked fun. After all, I was riding again for the first time in years after a serious injury so I was just grateful to be able to go and have the opportunity again despite my health problems. I'll always appreciate him inviting me. There's something about BMX, it makes friends out of strangers very easily, and, in my experience, brings out the best in people.

He told me to meet him at the bus station in the morning for a 45 minute ride to a town called Tababela, where we'd go to the home of his friend [José](https://www.fredducation.co/articles/person-jose-tababela-ecu/) to work on this mini ramp. So I rode over to the bus station and hopped on the bus with Diego and his girl friend, Anahi. Never had I seen a bus with speakers decorated like this with different types of fabric but now my Peruvian mother's obsession with upholstery wherever she can squeeze it in made sense. The ride over the mountain was fun. The driver handled that bus like there was a time he had to beat but nobody is complaining when you have somewhere to be. When we got to our stop, we hopped off and rode to José's house. Anahi jumped on Diego's bike and I got the photo. The property was peaceful with tons of plants and nooks to relax, as well as view of a canyon. Right in the middle of the backyard was the mini ramp Diego had come to help build up. While they did their thing, Anahi and I walked over to this tower that overlooked the canyon and took in the view and I snapped a few photos. It was a bit shaky in the wind so I came down faster than I climbed up. On the walk back Anahi pointed out this garden on the property and said this word I hadn't heard before, *cosechar*. She explained it meant to harvest from the plants by motioning with her hands. Now when I see or hear that word I think of her imitating the action of picking something from the earth with her fingers in an effort to explain to me what it means. Diego and José finished up the additions to the mini ramp they wanted to make so now came the fun part: riding it. I had told Diego about my recent comeback from injury, and so I think he knew I was nervous to ride despite me not saying anything. Like any good friend would, he insisted I ride the mini. I gave it a few carves and did a stall on the coping. It felt great. Life is full of little victories. Somewhere during this session Diego taught me how to say half pipe in Spanish: *medio tubo*. What a revelation, I was expecting some Spanglish. I took tons of photos of the guys riding during the session, I hope you enjoy them. 

We wrapped up the session and headed to get some food before the dirt park. I had some chicken, rice, beans, and lemonade, and nice charge of the batteries. Ecuador has some great food and I'm not going to get into it in this post but I will say that during lunch Diego, Anahi, and, José told me that I need to try [encebollado](https://es.wikipedia.org/wiki/Encebollado#:~:text=El%20encebollado%20es%20un%20caldo,lim%C3%B3n%20verde%20y%20con%20aj%C3%AD.) the next chance that I get. Well I did a few days later and they were right, it was delicious. When we got to the dirt park the boys did some maintenance work on the jumps before we started riding. I took the time to walk around a take some pictures, like the one above of a rouge cow that walked on the track. We started by riding the foam pit, something I've never done. It's all fun and games until the bar stabs you in the Johnson on the way down. While I was busy landing **nothing**, Diego was doing back flips because he's the man. *Volteras* is what he called them. I took some photos when he took it to the dirt jumps. I felt lucky to get the photo of him at the apex of the flip. A few weeks after I left Ecuador he did it out of a concrete lip at Parque Carolina in Quito. It got dark and the session came to a close. José gave us a lift to the bus stop, we got on, and took it back to Quito. When we got back we peddled together for a bit before we went our separate ways. I didn't know I wasn't going to see my dude again because I ended up breaking my hand a few days later and flying home for surgery, but that's a story for another post. I'll have to make my way back to Quito and catch a session with the homie for sure. Other than the good times, it's like going to language school. 

If you want to check out the man's riding, here is his [Instagram.](https://www.instagram.com/diegozunigafreire/)

**Comment by freddydrew** [comment]
[Making a gif](https://askubuntu.com/questions/648244/how-do-i-create-an-animated-gif-from-still-images-preferably-with-the-command-l)
